nft_ingester/src/api/api_impl.rs

```diff
@@ -128,22 +135,114 @@ where
         self.metrics.inc_requests(label);
         let latency_timer = Instant::now();
         let mut overall_status = Status::Ok;
-
-        //List of Checks
-        let mut check = Check::new("PostgresDB".to_string());
-
+        let mut last_sync_seq_pg_fungible: Option<u64> = None;
+        let mut last_sync_seq_pg_nft: Option<u64> = None;
+        let mut last_sync_seq_rocksdb_fungible: Option<u64> = None;
+        let mut last_sync_seq_rocksdb_nft: Option<u64> = None;
+
+        let mut pg_check = Check::new("PostgresDB".to_string());
+        let mut rocks_sync_divergence_check_fungible = Check::new("FungibleSync".to_string());
+        let mut rocks_sync_divergence_check_nft = Check::new("NonFungibleSync".to_string());
         if self.pg_client.check_health().await.is_err() {
             overall_status = Status::Unhealthy;
-            check.status = Status::Unhealthy;
-            check.description = Some(DasApiError::InternalDbError.to_string());
+            pg_check.status = Status::Unhealthy;
+            pg_check.description = Some(DasApiError::InternalDbError.to_string());
+            rocks_sync_divergence_check_fungible.status = Status::Unhealthy;
+            rocks_sync_divergence_check_fungible.description =
+                Some(DasApiError::InternalDbError.to_string());
```

Comment:
I would create a new error **InternalPgDbError** to specify more clear reason of why all sync checks are **Unhealthy**